src/backend/nodes/cypher_outfuncs.c

```diff
@@ -204,7 +204,6 @@ void out_cypher_node(StringInfo str, const ExtensibleNode *node)
 
     WRITE_STRING_FIELD(name);
     WRITE_STRING_FIELD(label);
-    WRITE_STRING_FIELD(parsed_label);
```

Comment:
Why were these removed? **parsed_label** was added into the struct for variable name lookup.